intermol/desmond/desmond_parser.py

```diff
@@ -1469,7 +1462,6 @@ def write_exclusions(self, moleculetype):
                                             toatom4 = atombonds[toatom1,j4]
                                             atomexclude.add(toatom4)
 
-                uniqueexclude = set(atomexclude)
```

Comment:
I removed a significant number of completely unused variables throughout the entire file. This is one of those.